### ISSUE TITLE
selectmenu ajaxified options refresh question

### ISSUE THREAD

**Issue by michaelpope** [fnagel/jquery-ui#201]
Hello!   Your work is incredible, and I hope you know that!  I tried my best to follow this thread to get the answer:

https://github.com/fnagel/jquery-ui/issues/17

but unfortunately, I couldn't quite grasp what the 'refresh' changes are and how i may implement them.  I have an old piece of ajax code that i use to update the college programs based on which campus the user selects.  When I activate the selectmenu, it must change the element so that it no longer speaks to my other piece of ajax that actually handle the change.  I implemented the ajax option, but it doesn't really get me there.  

Can you help guide me as to what you did to make the select menu refresh and provide various options, as you discuss in this previous thread?  Let me know if you need anything from me, and thank you for the rock solid coding work!

Michael

**Comment by fnagel** [comment]
With the refresh functionality you are able to init a selectmenu, change the original select and fire selectmenu() again to update the selectmenu widget. So just init your selectmenu, replace the the options within the original select via AJAX, and use "refresh". See Wiki (search for "Add / delete / modify options"): https://github.com/fnagel/jquery-ui/wiki/Selectmenu

Hope that helps!

ps: Youre always welcome to buy me a beer if you like my work on this effort ;-)
http://www.felixnagel.com/work/buy-me-a-beer/

**Comment by fnagel** [comment]
Please give some feedback!

**Comment by michaelpope** [comment]
Well,  I was not able to interpret your direction enough to make it work with my code, simply because I would have to learn everything about the code to understand your explanation.  That being said, the code seems great, and your thoughtful help with it was excellent, and I truly appreciate it, but I had to go with another option as I could not get the callback to do what I needed it to do.  This was due to my lack of knowledge about the code, not anything to do with the great work you have done.  Thank you very much for assistance, and I will come back to you in the future if I have an opportunity to try this code again....Thank you!
